### DISCUSSION TITLE
Unable to re-apply due to email address

### DISCUSSION THREAD

**Discussion by InternetRamen** [community/community#51379] [GitHub Education] [answered]
I'm unable to re-apply to the student pack because student pack requires me to use a student email. I have one, but it blocks emails from outside sources so I can't verify the email.

![image](https://user-images.githubusercontent.com/66806100/228618337-7cca4b6e-8c59-4af2-b174-3ba073b6b5bb.jpeg)

**Comment by samus-aran** [comment]
Hi @InternetRamen 👋 

Thank you for bringing this to our attention. We're passed this onto our internal team to investigate. When we have any updates we'll post it here. 👍

**Reply by InternetRamen** [reply]
Hi @samus-aran, thank you for the response. Do you have an update about the issue?

**Reply by samus-aran** [reply]
Hi @InternetRamen 👋 

We have updated the school details so you can proceed with your current email but because of this will request that you provide additional pieces of evidences. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. Remember we do accept multiple pieces of evidence, just include them in a single photo if possible e.g. a transcript or letter head alongside your Student ID.

Hope this helps! Let us know how you get along 👍

**Reply by InternetRamen** [reply]
Hi @samus-aran 

I managed to get past the email block, but I only have a picture of my student ID. The process doesn't let me upload that picture, saying that I need to use my device camera to post it. Is it possible that I can send the picture of the ID? I also tried uploading my school transcript with no avail.

**Reply by samus-aran** [reply, marked as answer]
Hi @InternetRamen 👋 

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. 

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. Try with your transcript again.

Let us know here if you need further assistance

**Reply by InternetRamen** [reply]
Thanks @samus-aran I submitted my transcript and received the benefits.

**Reply by samus-aran** [reply]
That's awesome @InternetRamen  ! 🎉 Thanks so much for coming back and sharing the update. This will help other community members seeking similar answers. 

What benefit are you most excited to try out first? 😄

**Reply by InternetRamen** [reply]
I love GitHub Copilot!
